web/src/routes/SurveyPages/SurveyListPage.tsx

```diff
@@ -1,95 +1,95 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 
-import { AxiosError, AxiosResponse } from 'axios';
+import { useQuery } from '@tanstack/react-query';
+import { AxiosError } from 'axios';
 import styled from 'styled-components';
 
-import axios from '../../api/axios';
-import { requests } from '../../api/request';
+import { fetchSurveyList } from '../../api/fetchFunctions';
+import ErrorPage from '../../components/ErrorPage';
 import Header from '../../components/Header';
 import { SurveyPreviewModal } from '../../components/Modal';
 import Pagination from '../../components/Pagination';
 import SurveyListSkeleton from '../../components/Skeleton/SurveyListSkeleton';
 import SurveyListTable from '../../components/SurveyListTable';
 import { useTheme } from '../../hooks/useTheme';
-import { SurveyAbstractResponse, SurveyPageResponse } from '../../types/response/Survey';
+import { SurveyPageResponse } from '../../types/response/Survey';
 
 const Container = styled.div`
   width: 100vw;
   height: 100vh;
   background-color: ${(props) => props.theme.colors.container};
 `;
 
+const ListContainer = styled.div``;
+
 export default function SurveyListPage() {
   const [theme, toggleTheme] = useTheme();
-  const [isLoading, setIsLoading] = useState<boolean>(false);
-  const [abortController, setAbortController] = useState<AbortController>(new AbortController());
-  const [surveys, setSurveys] = useState<SurveyAbstractResponse[]>([]);
   const [page, setPage] = useState<number>(1);
-  const [totalPages, setTotalPages] = useState<number>(0);
   const [previewModalOpen, setPreviewModalOpen] = useState<boolean>(false);
   const [selectedSurveyIndex, setSelectedSurveyIndex] = useState<number>(0);
 
-  const fetchSurveyList = async (abortSignal: AbortSignal): Promise<void> => {
-    setIsLoading(true);
-    try {
-      const request: AxiosResponse<SurveyPageResponse> = await axios.get<SurveyPageResponse>(
-        `${requests.getSurveyPage}${page}`,
-        { signal: abortSignal }
-      );
+  // FIXME: cacheTime and staleTime is appropriate?
+  const { data, isLoading, isError, error } = useQuery<SurveyPageResponse>(['surveyPage', page], fetchSurveyList, {
+    cacheTime: 9000,
+    staleTime: 60000,
+    retry: 1,
+    refetchOnWindowFocus: false,
+  });
 
-      setSurveys(request.data.surveys);
-      setTotalPages(request.data.totalPages);
-      setIsLoading(false);
-    } catch (error) {
-      const { name } = error as unknown as AxiosError;
-      if (name !== 'CanceledError') {
-        setSurveys([]);
-        setIsLoading(false);
-      }
-    }
-  };
+  if (isError) {
+    // TODO: 에러 종류에 따라서 다른 알림 표시
```

Comment:
And this could be refactored as well. As they all look the same

```typescript
<ErrorPage
  labelText="😥 SOME MESSAGE"
  buttonText="BUTTON TEXT"
  navigateRoute="/path/to/somewhere"
  theme={theme}
  toggleTheme={toggleTheme}
/>
```